templates/communities/communities.js

```diff
@@ -250,34 +250,36 @@ function verifyCommunity(community, doc) {
   }
 }
 
-function checkIfSoldOut(community, doc) {
+async function checkIfSoldOut(community, doc) {
   const mainSection = doc.querySelector('main > .section');
 
-  if (community.price !== 'Sold Out') return;
+  if (community.status !== 'Sold Out') return false;
 
   // Clear all content after breadcrumbs
-  const breadcrumb = mainSection.querySelector('.breadcrumbs');
+  const breadcrumb = mainSection.querySelector('.overview-wrapper');
   while (breadcrumb.nextSibling) {
     breadcrumb.nextSibling.remove();
   }
+  breadcrumb.remove();
```

Comment:
Is the breadcrumb getting removed? Can you give an example of a sold-out community page?